backend/src/app/rpc/commands/teams.clj

```diff
@@ -847,13 +935,16 @@
         (ex/raise :type :validation
                   :code :insufficient-permissions))
 
-      ;; First check if the current profile is allowed to send emails.
-      (when-not (eml/allow-send-emails? conn profile)
-        (ex/raise :type :validation
-                  :code :profile-is-muted
-                  :hint "looks like the profile has reported repeatedly as spam or has permanent bounces"))
+      ;; Check if the current profile is allowed to send emails.
+      (check-valid-email-muted conn profile true)
+
 
-      (let [cfg         (assoc cfg ::db/conn conn)
+      (let [requested      (->> (get-valid-requests-email conn team-id)
```

Comment:
should be `(into #{} (map :email) (get-valid-requests-email conn team-id))`

Reply:
fixed